File: Problemas/1985A - Creating Words/1985A_Creating_Words.py
# 1985A - Creating Words
# https://codeforces.com/problemset/problem/1985/A

# Input:
# 1) pruebas = Número de casos de prueba
pruebas = int(input())

# Itera sobre cada caso de prueba
for indice_prueba in range(0,pruebas):

    # Input:
    # 2) a, b = Strings cuyas letras se intercambiarán
    a, b = map(str, input().split())

    # Intercambia la primera letra de los dos strings entre sí
    new_a = b[0] + a[1:]
    new_b = a[0] + b[1:]

    # Imprime los nuewvos dos strings
    print("{} {}".format(new_a, new_b))

File: Problemas/1985A - Creating Words/1985A_Creating_Words.cpp
// 1985A - Creating Words
// https://codeforces.com/problemset/problem/1985/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) pruebas = Número de casos de prueba
    int pruebas;
    cin >> pruebas;

    // Itera sobre cada caso de prueba
    for(int indice_prueba = 0; indice_prueba < pruebas; indice_prueba++) {

        // Input:
        // 2) a, b = Strings cuyas letras se intercambiarán
        string a, b;
        cin >> a >> b;

        // Extrae la 1° letra de ambos strings
        char first_letter_a = a[0];
        char first_letter_b = b[0];

        // Intercambia la 1° letra entre ambos strings
        a[0] = first_letter_b;
        b[0] = first_letter_a;

        // Imprime los strings con las letras intercambiadas
        cout << a << " " << b << endl;
    }

    return 0;
}